gamebook/scene.py

```diff
@@ -34,20 +34,22 @@ def show_options(self):
         
         return choice_list
 
-    def get_option_from_list(self,option_to_find):
-        for opt in self.options:
-            if option_to_find == opt:
-                option_to_return = option_to_find
-        
+    def get_option_by_user_input(self,user_input_recieved):
+        option_to_return = Option("",[],[])
+        for option in self.options:
+            for user_input in option.user_input:
+                if user_input == user_input_recieved:
+                    option_to_return = option
```

Comment:
why use `option_to_return`? just return it here, no?

Reply:
you are right :upside_down_face: